src/pages/Setting/Setting.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Route } from 'react-router-dom';
+import { Header, SplitType } from '@/components';
+import { useSurveyForm } from '@/hooks';
+import style from './setting.module.scss';
+
+const SURVEY_STATE_KEY = {
+  SPLIT_TYPE: 'splitType',
+  AUDIO_COACH: 'audioCoach',
+  SPEED: 'speed',
+  EXPLANATION: 'explanation',
+};
+
+const { s_componentContainer } = style;
+
+const Setting = () => {
+  const { surveyState, setSurveyStateByKey } = useSurveyForm();
+
+  const handleCloseCurrentPage = () => {
+    window.close();
+  };
+
+  return (
+    <>
+      <Header handleClickBackButton={handleCloseCurrentPage} />
+      <form>
+        <div className={classNames(s_componentContainer)}>
+          <h1 className={classNames('s_a11yHidden')}>분할타입 변경</h1>
+          <Route path="/setting/split-type">
```

Comment:
해당 라우트도 상수로 관리하면 좋을거 같아요 아래와 같은 방법도 제안 드립니다!
```suggestion
          <Route path=`${SETTING_PAGE}/split-type`>
          
          // OR 상수 정의할때 한번에
          
          const SETTING_SPLIT_TYPE_PAGE = '/setting/split-type';
```